lib/resources/r4/organization.yaml

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
Take a look at FinancialTransactions URLs and then yours. You should update yours accordingly :)

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/326/commits/884311d2afad568b4d627e742466974d702778a3